spec/product/reports_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe 'YAML reports' do
+  let(:report_dirs) { [REPORTS_FOLDER, "#{TIMELINES_FOLDER}/miq_reports"] }
```

Comment:
I dislike how we include UiConstants globally.  My first question is where is this constant defined.  Can we explicitly use UiConstants:: REPORTS_FOLDER and the other one here?